LRRServer/migrations/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
I wasn't sure what why the migration code was needed here, could you expand in a comment or the README?

Is this a temporary/one-off measure, since we have existing data stored as CSVs, or do you see this being part of an ingestion pipeline run more regularly?

Reply:
that's a generated readme